app/views/appointment_attempts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Book an appointment</h1>
+
+<%= form_for @appointment_attempt, layout: :basic do |f| %>
+  <%= f.text_field :first_name, class: 't-first-name' %>
+  <%= f.text_field :last_name, class: 't-last-name' %>
+  <%= f.date_field :date_of_birth, class: 't-date-of-birth', data: { module: 'CustomerAge', output: '.customer-age' } %>
+  <div class="bg-info customer-age">
```

Comment:
Tiny thing but we try to prefix javascript targets with 'js-'  which then aren't used for styling. Just means we can refractor css without breaking js.